src/utils.ts

```diff
@@ -123,6 +128,110 @@ export function extensionConfigurationSet<
   return getConfiguration().update(key, value, target);
 }
 
+export function findLanguageServer(client: LanguageServerClient): string | null {
+  const platform = os.platform();
+  const executableName = client.executableName
+  if (platform != "win32" && platform != "darwin")
```

Comment:
Some of this logic seems specific to your language server. Might try to find a way to move this into the abstract class somehow?

Reply:
Maybe some computed properties that returns all supported OS/Arch combinations?

Reply:
Perhaps. I am thinking of a `LanguageServerClient::find()` function or some sort. You could mark the function as not abstract, and then use the computed property like you said. Might not need each subclass to write their own impl that way.